src/main/java/io/vertx/ext/jdbc/impl/actions/JDBCClose.java

```diff
@@ -28,13 +29,23 @@
  */
 public class JDBCClose extends AbstractJDBCAction<Void> {
 
-  public JDBCClose(Vertx vertx, SQLOptions options, ContextInternal ctx) {
+  private final PoolMetrics metrics;
+  private final Object metric;
+
+  public JDBCClose(Vertx vertx, SQLOptions options, ContextInternal ctx, PoolMetrics metrics, Object metric) {
     super(vertx, options, ctx);
+    this.metrics = metrics;
+    this.metric = metric;
```

Comment:
Naming a bit confusing. Not sure there is anything better.

Maybe adding Javadoc on the fields would help to understand the differences between ’metric’ and ’metrics’.